### PR TITLE
UI and API integeration for login and register pages

### LINKED ISSUE #41: Login and Register Page
Create UI and integerate with the user APIs

### PR DESCRIPTION
**Pull request by bodhisha**
Resolves #41 

**Changes**
- Add UI for login and register
- Integrate with API

### CODE REVIEW

**Review by DebugAgrawal — APPROVED** [review]

File: frontend/src/Components/Account/Login.js [hunk 1]
```diff
@@ -1,5 +1,103 @@
...
+                Email
+              </label>
+              <input
+                aria-label="user name"
+                name="email"
+                value={form.email}
+                type="email"
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="Email"
```
Comment (aitikgupta) [inline frontend/src/Components/Account/Login.js]: "Email..."? XD

File: frontend/src/Components/Account/Login.js [hunk 2]
```diff
@@ -1,5 +1,103 @@
...
+                Password
+              </label>
+              <input
+                aria-label="Password"
+                name="password"
+                type="password"
+                value={form.password}
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="******************"
```
Comment (aitikgupta) [inline frontend/src/Components/Account/Login.js]: ig 5 *s are enough, there are almost 20 *s XD

File: frontend/src/Components/Account/Login.js [hunk 3]
```diff
@@ -1,5 +1,103 @@
...
+              />
+            </div>
+
+            {formError && (
+              <div className="text-red-400 font-semibold">
+                Invalid Credentials
```
Comment (aitikgupta) [inline frontend/src/Components/Account/Login.js:76]: Will be solved by https://github.com/Open-Sourced-Olaf/SocioMark/issues/32, 👍🏼 for now

File: frontend/src/Components/Account/Register.js [hunk 4]
```diff
@@ -1,5 +1,159 @@
...
+                Email
+              </label>
+              <input
+                aria-label="user name"
+                name="email"
+                value={form.email}
+                type="email"
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="Email"
```
Comment (aitikgupta) [inline frontend/src/Components/Account/Register.js]: Same^

File: frontend/src/Components/Account/Register.js [hunk 5]
```diff
@@ -1,5 +1,159 @@
...
+                Password
+              </label>
+              <input
+                aria-label="Password"
+                name="password"
+                type="password"
+                value={form.password}
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="******************"
```
Comment (aitikgupta) [inline frontend/src/Components/Account/Register.js]: Same^ XD

File: frontend/src/Components/Account/Register.js [hunk 6]
```diff
@@ -1,5 +1,159 @@
...
+                type="description"
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="Describe Yourself..."
+              />
+            </div>
+
+            {formError && (
+              <div className="text-red-400 font-semibold">
+                Invalid Credentials
```
Comment (aitikgupta) [inline frontend/src/Components/Account/Register.js:132]: Sign Up doesn't need to have Invalid Credentials, we can think about it once we have a Confirm Password? Probably best will be handled by validators https://github.com/Open-Sourced-Olaf/SocioMark/issues/24

**Review by aitikgupta — APPROVED** [review]
Awesome 🎉

File: frontend/src/Components/Account/Login.js (hunk 2 above)
Comment (bodhisha) [reply frontend/src/Components/Account/Login.js]: xD Only 18 *s

**Review by aitikgupta — APPROVED** [review]
LGTM 🌮

**Pull request merged**